### ISSUE TITLE
A11y_AzureDataStudioSQL_PublishProjects_Generate Scripts(Terminal)_Keyboard: Horizontal scrollbar is not scrollable using keyboard when screen is zoomed to 200% 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#22423]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev3;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win11;#FTP;#DesktopApp;#WCAG2.1.1;#AINotInScope;#Zoom;

**## Prerequisite**:
Set the resolution to 1280*768
and change text sizing to 200%

## Environment Details:
Application Name: Azure Data Studio
Windows Version: Win10

## Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an press shit+f10 on any created project.
5. Tab till "Publish Project" and hit enter
6. Hit on Generate script and navigate to task
7. Check if scrollbar is getting scrolled using keyboard

## Actual:
Horizontal scrollbar is not scrollable using keyboard when screen is zoomed to 200%

## Expected:
Horizontal scrollbar should bet scrollable using keyboard when screen is zoomed to 200%

## User Impact:
Keyboard only users will face difficulty in getting the information

## Attachment:
[Video 3-23 at 16.58.webm](https://user-images.githubusercontent.com/24580570/227190887-552baafe-d726-49d7-957e-13001efa9dfb.webm)

**Comment by fsteffi** [comment]
bug reopened from https://github.com/microsoft/azuredatastudio/issues/15098

**Comment by kisantia** [comment]
I tried setting the keyboard shortcuts, as mentioned in https://github.com/microsoft/vscode/issues/121132, and it worked for the connections and projects view, but wasn't working for Tasks. Have to investigate why the same shortcuts aren't working for Tasks since I thought that was also using the treeview
